README.md

```diff
@@ -1,3 +1,19 @@
+## Deprecation Notice
+
+With the release ``0.3.0`` of the ``astro-provider-databricks`` package, this provider stands deprecated and will
+no longer receive updates. We recommend migrating to the official ``apache-airflow-providers-databricks>= 6.8.0`` for the latest features and support.
+For the operators and sensors that are deprecated in this repository, migrating to the official Apache Airflow Databricks Provider
+is as simple as changing the import path in your DAG code as per the below examples.
+
+| Previous import path used (Deprecated now)                                    | Suggested import path to use                                                                                |
+|-------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| `from astro_databricks.operators.notebook import DatabricksNotebookOperator`  | `from airflow.providers.databricks.operators.databricks import DatabricksNotebookOperator`                  |
```

Comment:
```suggestion
| <code>`from astro_databricks.operators.notebook import DatabricksNotebookOperator`</code>  | <code>`from airflow.providers.databricks.operators.databricks import DatabricksNotebookOperator`</code>                  |
```
Shall we verify if this works and see how it renders?

Reply:
The current text renders fine if we look at the rich diff on the PR review files page

<img width="1372" alt="Screenshot 2024-08-07 at 11 44 18 AM" src="https://github.com/user-attachments/assets/b3e49cad-5712-4ce6-9bee-d0c9a3f64a01">